packages/ts/src/core/xy-component/index.ts

```diff
@@ -87,4 +84,12 @@ export class XYComponentCore<
     const yAccessors = (isArray(config.y) ? config.y : [config.y]) as NumericAccessor<Datum>[]
     return getExtent(data, ...yAccessors)
   }
+
+  getAriaDescription (): string {
+    const xDataExtent = this.getXDataExtent()
+    const yDataExtent = this.getYDataExtent(true)
+    let description = `Extent of ${ScaleDimension.X} is ${xDataExtent[0]?.toFixed(2)} to ${xDataExtent[1]?.toFixed(2)} `
```

Comment:
`Extent of ${ScaleDimension.X} is ${xDataExtent[0]?.toFixed(2)} to ${xDataExtent[1]?.toFixed(2)} ` → 
`Extent of the ${ScaleDimension.X} dimension spans from ${xDataExtent[0]?.toFixed(2)} to ${xDataExtent[1]?.toFixed(2)}`

If the extent is `undefined`, the description will say "undefined to undefined" . I think that edge case has to be handled separately, and say something like: "The extent of the X dimension is undefined"